### DISCUSSION TITLE
Force reconvert

### DISCUSSION THREAD

**Discussion by PiousVenom** [openaudible/openaudible#1465] [General]
Is it possible to force a reconvert on the file? For instance, I've got an M4B file that when I import into iTunes says its 700+ hrs long. That is obviously incorrect. Now, I converted this file several versions ago, and I'm wanting to do another conversion to see if that works.

**Comment by openaudible** [comment]
Not easily. You'll need to manually delete the file. Then use Controls: Refresh Files. Then re-encode.
